tests/components/test_component_manifold_extraction.cpp

```diff
@@ -0,0 +1,207 @@
+// #include <catch2/catch.hpp>
+#include <catch2/catch_test_macros.hpp>
+#include <wmtk_components/delaunay/internal/delaunay_2d.hpp>
+#include <wmtk_components/delaunay/internal/delaunay_3d.hpp>
+#include <wmtk_components/manifold_extraction/internal/Manifold-extraction.hpp>
+#include <paraviewo/VTUWriter.hpp>
+#include <igl/adjacency_matrix.h>
+#include <igl/connected_components.h>
+
+
+auto tagassign(size_t nb_triangles, double prob) -> std::vector<size_t>{
```

Comment:
I guess you just copy&pasted your code into this file but you should use the new attributes for basically everything. Of course, first you need to use a `TriMesh` here but this is also something you should do.